packages/transaction-log/src/components/common/animated.table.row.component.tsx

```diff
@@ -0,0 +1,39 @@
+import {TableRow} from '@mui/material'
+import {useRef, useEffect, useState} from 'react'
+
+const sleep = ms => new Promise(resolve => setTimeout(resolve, ms))
+
+export const AnimatedTableRow = ({
+  children,
+  initialColor,
+  finalColor,
+  onClick
+}) => {
+  const ref = useRef(null)
+  const [completed, setCompleted] = useState(false)
+
+  useEffect(() => {
```

Comment:
Does this cause an infinite loop of some sort? 🤔

`useEffect` runs whenever `completed` is modified. Yet `useEffect` also causes `completed` to be modified.

Reply:
Thanks, an infinite loop is not occuring but I removed it just to be safe, and the behaviour is still the same.